pipeline/src/lib.rs

```diff
@@ -37,6 +37,11 @@ pub use conclusion::{
 };
 pub use config::{ConclusionFeedSource, Config};
 
+pub const CONCLUSION_FEED_TABLE: &str = "ceramic.v0.conclusion_feed";
+pub const DOC_STATE_TABLE: &str = "ceramic.v0.doc_state";
+pub const DOC_STATE_MEM_TABLE: &str = "ceramic._internal.doc_state_mem";
+pub const DOC_STATE_PRESISTENT_TABLE: &str = "ceramic._internal.doc_state_persistent";
```

Comment:
PRESISTENT -> PERSISTENT 😄

Reply:
I can't type in all caps apparently, thanks I'll get this fixed.